### ISSUE TITLE
Remove references to split code

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#3713]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)| @thijstriemstra reported|
|-|-|
|Trac ID|trac#3713|
|Type|defect|
|Created|2009-03-30 18:46:05Z|
|Branch|https://github.com/twisted/twisted/tree/ref-split-code-3713|

The docstring of the [twisted/protocols/__init__.py](twisted/protocols/__init__.py) module contains a reference to code that was removed with [#3581](https://github.com/twisted/twisted/issues/3581). Also some left in [twisted/protocols/xmlstream.py](twisted/protocols/xmlstream.py)

<details><summary>Searchable metadata</summary>

```
trac-id__3713 3713
type__defect defect
reporter__thijs thijs
priority__normal normal
milestone__ 
branch__branches_ref_split_code_3713 branches/ref-split-code-3713
branch_author__thijs thijs
status__closed closed
resolution__fixed fixed
component__core core
keywords__documentation documentation
time__1238438765000000 1238438765000000
changetime__1238937155000000 1238937155000000
version__None None
owner__ 
cc__thijs
```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)|@thijstriemstra set status to `assigned`|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_2"></a>|@thijstriemstra commented|
|-|-|

(In [26543]) Branching to 'ref-split-code-3713'

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_3"></a>|@thijstriemstra commented|
|-|-|

(In [26544]) Remove old part of docstring and get rid of deprecated t.p.xmlstream refs [#3713](https://github.com/twisted/twisted/issues/3713)

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_4"></a>|@thijstriemstra commented|
|-|-|

(In [26545]) Update the split faq, refs [#3713](https://github.com/twisted/twisted/issues/3713)

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_5"></a>|@thijstriemstra removed owner<br>@thijstriemstra set status to `new`|
|-|-|

Up for review.

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_6"></a>|@exarkun set owner to @thijstriemstra|
|-|-|

`twisted.test.test_split_compat` :(

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_7"></a>|@thijstriemstra commented|
|-|-|

(In [26590]) Remove old test refs [#3713](https://github.com/twisted/twisted/issues/3713)

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)|@thijstriemstra removed owner|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_9"></a>|@exarkun set owner to @thijstriemstra|
|-|-|

Looks good, please merge.  I sure hope there's nothing else hiding. ;)

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_10"></a>|@thijstriemstra set status to `closed`|
|-|-|

(In [26647]) Merge ref-split-code-3713: remove reference to split code

Author: thijs
Reviewer: exarkun
Fixes: [#3713](https://github.com/twisted/twisted/issues/3713)

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|
